scripts/hydra

```diff
@@ -423,6 +424,13 @@ function check_if_should_run_update() {
     load_scripts
     run_migrations
   fi
+
+  local ref_type=$(jq -r '.ref_type // empty' "$ROOT_PATH/euclid.json")
```

Comment:
will this handle branch names that include slashes (eg. `feature/fee-transaction`)?

Reply:
Yes, you can take a look in get-information function where we remove special chars to generate the TESSELLATION_VERSION_NAME